java/short_url_generator/README.md

```diff
@@ -0,0 +1,56 @@
+# 💻 Get Short URL
+
+A Java Cloud Function for generating a Short URL using [tinyurl](https://tinyurl.com/app) and [bitly](https://bitly.com/)
+
+_Example input:_
+
+```json
+{
+    "provider": "tinyurl",
+    "url": "https://google.com"
+}
+```
+
+Supported providers: tinyurl, bitly
+
+_Example output:_
+
+
+```json
+{
+    "success": true,
+    "url": "https://tinyurl.com/dck67zxk"
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
+
+**apiKey** - This must be passed within 'variables' section of the request. This API key would be the access token used by the provider for generating a short URL.
```

Comment:
Let's please separate this into 2 API keys - one for Bitly, one for TinyURL. That will allow user use both at the same time.

Reply:
We could call them `TINYURL_API_KEY` and  `BITLY_API_KEY`.